src/Spanner/composer.json

```diff
@@ -5,9 +5,9 @@
     "minimum-stability": "stable",
     "require": {
         "ext-grpc": "*",
-        "google/cloud-core": "^1.4",
-        "google/gax": "^0.10",
-        "google/proto-client-php": "^0.13"
+        "google/cloud-core": "^1.6",
```

Comment:
To allow travis to pass (since it is running a check to determine if components can be installed properly) we will need to relax this to a currently tagged release.